CONTRIBUTING.md

```diff
@@ -50,37 +60,24 @@ This will add a folder inside `src/components` with the name `CoolCatsAndKittens
 -   `CoolCatsAndKittens.spec.tsx` is the place to write your tests 🔬
 -   `CoolCatsAndKittens.stories.tsx` is for your Stories inside Storybook 📄
 
-## Opening a PR
-
-Once you're happy with your new component, it's time to open a new PR. Push your branch to GitHub as a start:
-
-```shell
-git push -u origin feat/cool-cats-and-kittens
-```
+## Building Packages
 
-and click the link provided by the git CLI.
+The Packages in the Monorepo are built independently.
+To build the packages, run `pnpm build` in the respective package directory.
 
-### Naming your PR
+Alternatively you can run the following scripts in the root directory.
 
-The Pull Requests basically follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/).
-
-The name of your PR should be:
-
-```
-<type>[optional scope]: <description>
-```
-
-Following the example from above, it would look like this:
-
-```
-feat: Add CoolCatsAndKittens Component
-```
+-   `pnpm build:components`
+-   `pnpm build:fondue`
+-   `pnpm build:icons`
+-   `pnpm build:charts`
 
-Breaking Changes can be declared with a exclamation mark (!) after the type.
+## Releasing Packages
 
-### GitHub Actions
+We utilize [Changesets](https://github.com/changesets/changesets) to manage our releases.
 
--   GitHub automatically deploys Storybook with your new changes to Netlify and adds a Preview link to the PR.
--   SonarCloud checks the code base for its quality.
--   Cypress tests are run to make sure, you're not breaking anything.
+To bump the version of a Package, run `pnpm release` in the root directory and follow the prompts to select your package and version bump.
```

Comment:
Could you add a paragraph about the description? 
The description should be prefixed with "feat:" / "fix:" / ..., be easily readable and understandable

Reply:
added